assets/js/blocks/checkout/block.tsx

```diff
@@ -28,6 +30,8 @@ import CheckoutOrderError from './checkout-order-error';
 import { LOGIN_TO_CHECKOUT_URL, isLoginRequired, reloadPage } from './utils';
 import type { Attributes } from './types';
 import { CheckoutBlockContext } from './context';
+import { hasNoticesOfType } from '../../utils/notices';
```

Comment:
This would actually be a selector in `@wordpress/data` land, but again we don't control the data store, but it could definitely be worth improving the data store and adding this to my PR (https://github.com/WordPress/gutenberg/pull/39940)